### ISSUE TITLE
Help me pls

### ISSUE THREAD

**Issue by FeanlyVer** [jpw1991/chebs-necromancy#248]
# Description

I'm an Xbox user,i'm unable to play normally due to updates from Valheim(Updated to 0.217.37),It keeps reporting errors when I enter the main menu.At the first I think it's a problem with the ’Jotunn‘ mod,But after testing, I found that J could be loaded alone, and it started reporting errors until I added the ’cheb‘ mod.

## Attachments Checklist

- [
[LogOutput.log](https://github.com/jpw1991/chebs-necromancy/files/13623533/LogOutput.log)
] Log file
- [
[com.chebgonaz.ChebsNecromancy.zip](https://github.com/jpw1991/chebs-necromancy/files/13623534/com.chebgonaz.ChebsNecromancy.zip)
] Config file

## Screenshots

![1](https://github.com/jpw1991/chebs-necromancy/assets/79962144/94b19b2b-d6f0-40aa-bb52-e62161098c00)
![2](https://github.com/jpw1991/chebs-necromancy/assets/79962144/03ec9abf-d73a-4480-b392-4944c4683953)

**Comment by jpw1991** [comment]
Hello, I'll look into it later today and get back to you. Thanks for opening the issue and providing your log etc

**Comment by jpw1991** [comment]
I think your Valheim version is too new for the mod. Current is `0.217.31 (network version 18)` and you're running `0.217.37 (network version 20)`

**Comment by jpw1991** [comment]
Yes, that's right. Are you able to downgrade? I don't support PTB versions of Valheim - only the latest official releases.

**Comment by FeanlyVer** [comment]
> Yes, that's right. Are you able to downgrade? I don't support PTB versions of Valheim - only the latest official releases.


Okay, I will be try. Thanks for your answer.
